app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+
+import ResultsRow from "experimenter-rapid/components/experiments/ResultsRow";
+import { RESULT_COLUMN_TIPS } from "experimenter-rapid/components/experiments/tooltips";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { AnalysisPoint, ExperimentData } from "experimenter-types/experiment";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
```

Comment:
Duplicate of definition above?

https://github.com/mozilla/experimenter/pull/3543/files#diff-6939e957e4c934ca650de0c2038fd1b6R6-R10